index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Survey Form</title>
+  <style>
+    body {
+      font-family: Arial, sans-serif;
+      margin: 0;
+      padding: 0;
+    }
+    
```

Comment:
-Please, I suggest installing the linters; it will be helpful to detect some style errors before you update your project changes.
-Apart, you currently do have not a professional readme file, however, this is an important requirement in future projects you will build here at Microverse.